content/docs/services/intro.md

```diff
@@ -8,23 +8,26 @@ title: Introduction
 weight: -1
 ---
 
-You can expand the functionality of your cloud.gov application by making use of services. Before your application can use a service, the service must be provisioned, and credentials for using the service must be supplied to your application. 
+You can expand the functionality of your cloud.gov application by making use of services. Before your application can use a service, you must provision the service, and supply the credentials for using the service to your application. 
```

Comment:
Not a blocker, but in my opinion you don't need a comma between "you must provision the service, and supply the credentials for using the service to your application."